### DISCUSSION TITLE
Production build error :(

### DISCUSSION THREAD

**Discussion by apostle51** [intlify/vue-i18n#1954] [Q&A]
I’ve implemented Vue 3 with Vite and integrated vue-i18n for internationalization . Everything works perfectly in my local development environment, including language files and I18n functionality, without any errors.

However, when I deploy the project to the server and run yarn build, the build process completes without any issues, but when I load the app in the browser, I encounter several errors. The errors include multiple instances of the following:

`SyntaxError: 23 (at @intlify-2rgc3MTt.js:sourcemap:112:17)
    at createCompileError (@intlify-2rgc3MTt.js:sourcemap:112:17)
    at createCoreError (@intlify-2rgc3MTt.js:sourcemap:1646:10)
    at resolveLocale (@intlify-2rgc3MTt.js:sourcemap:1669:13)
    at getLocale (@intlify-2rgc3MTt.js:sourcemap:1649:67)
    at translate (@intlify-2rgc3MTt.js:sourcemap:2657:18)
    at vue-i18n.mjs:517:48
    at wrapWithDeps (vue-i18n.mjs:462:19)
    at Proxy.t (vue-i18n.mjs:517:16)
    at Header.vue:413:3
    at Object.renderFnWithContext [as default] (runtime-core.esm-bundler.js:694:13)`
    
    I have followed all necessary steps, including adding the vue-i18n plugin to vite.config.js, configuring language files correctly, and optimizing the build process by turning off minification and enabling sourcemaps for debugging.

The project works without issues locally but doesn’t function correctly when deployed on the server.

Any insights or suggestions would be highly appreciated.

This message explains the problem clearly and provides the context that everything works locally but breaks on the server.

**Comment by IvalinV** [comment]
@apostle51 Almost certainly I'm way too late to answer, but I've recently stumbled upon the same issue with `i18n` package in my `Vue 3` project, the problem in my case was that when my project was build for production the `locale.value` used from the package was receiving `undefined` which caused the issue. Make sure that a proper value is passed to the package.

P.S In production, `vue-i18n` strips the human-readable message and the error message becomes the bare numeric code from `@intlify/core-base`. Code 23 = "cannot support locale type",
